### PR TITLE
Bowling gui

### PR DESCRIPTION
**Pull request by dbowling21**
Updated the gridbaglayout of GUI to handle resizing and future additions

### CODE REVIEW

**Comment by dbowling21** [comment]
All conflicts are hopefully resolved. It's possible a method was partially auto-deleted by git but there wasn't enough info to know which.

**Review by GalacticWafer — APPROVED** [review]
This is much cleaner since starting to use methods for a lot of the gui creation, and I really approve of the setNewModel method. We will have to modify other code to use this method, so it will need to be public soon (so that we can set information from Analytics).

File: src/main/java/GUI.java [hunk 1]
```diff
@@ -108,19 +109,11 @@ private String checkConnection() throws SQLException {
...
-		north.add(status);
-		frame.add(north, BorderLayout.NORTH);
-		frame.add(east, BorderLayout.EAST);
-		frame.add(west, BorderLayout.WEST);
-		frame.add(south, BorderLayout.SOUTH);
-		frame.add(center, BorderLayout.CENTER);
-		frame.setLocationRelativeTo(null);
-		frame.setVisible(true);
-		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+	private void refresh() throws SQLException {
```
Comment (Hamadesigns) [inline src/main/java/GUI.java:112]: refresh method needs to be changed to public, but overall great job.

**Review by Hamadesigns — APPROVED** [review]

**Comment by dbowling21** [comment]
Definitley good items for the next sprint to look at

**Pull request merged**